### PR TITLE
V4.0 plugin Welcome

### PR DESCRIPTION
**Pull request by clickwork-git**
- Modification of the code from line 23 to add translation for "Welcome".
- Translations

### CODE REVIEW

File: bl-plugins/welcome/languages/en.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
...
-	"plugin-data":
-	{
-		"name": "Welcome",
-		"description": "Shows a greeting message in the dashboard."
-	}
-}
\ No newline at end of file
+    "plugin-data":
+    {
+        "name": "Last pages",
```
Comment (dignajar) [inline bl-plugins/welcome/languages/en.json:4]: this files is incorrect, looks like is from the plugin "last pages'

Comment (clickwork-git) [reply bl-plugins/welcome/languages/en.json]: Sorry! It should be the right one now.

Comment (clickwork-git) [reply bl-plugins/welcome/languages/en.json]: Sorry! It should be the right one now.

**Comment by dignajar** [comment]
thank you

**Pull request merged**